test/RToken.test.ts

```diff
@@ -1372,17 +1385,44 @@ describe(`RTokenP${IMPLEMENTATION} contract`, () => {
       await token3.connect(addr1).approve(rToken.address, initialBal)
 
       // Issue rTokens
-      await rToken.connect(addr1).issue(issueAmount)
+      await expect(rToken.connect(addr1)['issue(uint256)'](instantIssue)).to.emit(
+        rToken,
+        'Issuance'
+      )
 
-      // Attempt to vest
-      await expect(rToken.vest(addr1.address, 1)).to.be.revertedWith('not ready')
-      await advanceBlocks(5)
+      expect(await rToken.balanceOf(addr1.address)).to.equal(instantIssue)
+    })
 
-      // Issue rTokens
-      await expect(rToken.connect(addr1).issue(instantIssue)).to.emit(rToken, 'IssuanceStarted')
+    it('Should allow issuances to a different account', async function () {
+      const instantIssue: BigNumber = MIN_ISSUANCE_PER_BLOCK.sub(1)
+      const issueAmount: BigNumber = MIN_ISSUANCE_PER_BLOCK.mul(3)
 
-      // Should vest now
-      await expect(await rToken.vest(addr1.address, 1)).to.emit(rToken, 'IssuancesCompleted')
+      // Provide approvals
+      await token0.connect(addr1).approve(rToken.address, initialBal)
+      await token1.connect(addr1).approve(rToken.address, initialBal)
+      await token2.connect(addr1).approve(rToken.address, initialBal)
+      await token3.connect(addr1).approve(rToken.address, initialBal)
+
+      // Can instantly issue rTokens to another account
+      expect(
+        await rToken
+          .connect(addr1)
+          .callStatic['issue(address,uint256)'](addr2.address, instantIssue)
+      ).to.eq(instantIssue)
+      await expect(
+        rToken.connect(addr1)['issue(address,uint256)'](addr2.address, instantIssue)
+      ).to.emit(rToken, 'Issuance')
+
+      // Cannot vest RTokens before they're ready
+      expect(
+        await rToken.connect(addr1).callStatic['issue(address,uint256)'](addr2.address, issueAmount)
+      ).to.eq(0)
+      await rToken.connect(addr1)['issue(address,uint256)'](addr2.address, issueAmount)
+      await expect(rToken.vest(addr2.address, 1)).to.be.revertedWith('not ready')
+
+      // Should vest now after 5 blocks
+      await advanceBlocks(5)
+      await expect(await rToken.vest(addr2.address, 1)).to.emit(rToken, 'IssuancesCompleted')
     })
```

Comment:
I think it's worth it to add a test that does `cancel()`. I know it's fairly obvious cancellation should work since we put it in the recipient's queue, but I still think it's worth it.

Reply:
Agreed, the latest push tests issuing to another account and that other account canceling. Will highlight it once the code is up!